src/main/plugins/browser-bookmarks-plugin/firefox-bookmark-repository.ts

```diff
@@ -0,0 +1,61 @@
+        type: IconType.SVG
+    };
+
+    private readonly userDataFolderPath: string;
+
+    constructor(userDataFolderPath: string) {
+        this.userDataFolderPath = userDataFolderPath;
+    }
```

Comment:
This could be simplified to this:

```typescript
constructor(private readonly userDataFolderPath: string) {}
```

Reply:
Of course. I'm glad you [changed your mind](https://github.com/oliverschwendener/ueli/pull/250#discussion_r358374508). 😉

Reply:
Haha, yeah sorry about that one 😅